.github/CODEOWNERS

```diff
@@ -0,0 +1,4 @@
+@alexellis
```

Comment:
I would keep this as me for now given that I'm not aware of any code commits

Reply:
Will change 👍

Reply:
Done ✔️